cfgov/unprocessed/css/enhancements/base.less

```diff
@@ -1,28 +1,592 @@
+// BEGIN enhancement for normalize-legacy-addon
+
+/*
+ * Corrects list images handled incorrectly in IE 7.
+ */
+
+nav ul,
+nav ol,
+nav ul ul,
```

Comment:
You only need this one right? The other three are just excess that should probably be removed to avoid duplication.